panel/models/event-to-object.ts

```diff
@@ -25,6 +25,11 @@ SOFTWARE.
 export function serializeEvent(event: any): any {
   const data: any = {};
 
+  // support for CustomEvents: the whole `detail` object is serialized
+  if (Object.getPrototypeOf(event).initCustomEvent !== undefined) {
```

Comment:
Any reason to use this specific detection code, instead of `event instanceof CustomEvent`?

Reply:
@mattpap yep, more simple, but it will be also true for `Event` instance.

Reply:
… I mean: there's nothing wrong on supporting also `Event`, but it's the general design of this module seems to want to limit properties read from an event and I didn't wanted to go against this